pdfminer/pdffont.py

```diff
@@ -403,9 +403,9 @@ def __init__(self, name, fp):
         return
 
     def create_unicode_map(self):
-        if 'cmap' not in self.tables:
+        if b'cmap' not in self.tables:
             raise TrueTypeFont.CMapNotFound
-        (base_offset, length) = self.tables['cmap']
+        (base_offset, length) = self.tables[b'cmap']
```

Comment:
Presumably this never worked before because all elements of `self.tables` are bytes?

Reply:
I made a compatible version for both str and byte originally, but I found it did not break any test if only bytes. So I kept it simple.